Healthy/Classes/Modules/Login/LoginViewController.swift

```diff
@@ -33,15 +35,22 @@ final class LoginViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
         configureAppearance()
+        bindTextFieldsChanges()
+        bindLoadingIndicator()
+        bindErrorMessage()
+        bindButtonState()
+        bindLoginStatus()
```

Comment:
I would suggest having two methods, one for inputs and another one for outputs. 
`configureViewModelInputs()`
`configureViewModelOutputs()`

Since this will get more complex in the `viewDidLoad` once we have more view logic added, navigation controller, other appearance configurations, ..etc.